gradle.properties

```diff
@@ -4,8 +4,11 @@ kotlin.incremental.multiplatform=true
 kotlin.mpp.stability.nowarn=true
 kotlin.native.ignoreDisabledTargets=true
 
+# gradle
+org.gradle.jvmargs=-Xmx2G -XX:MaxMetaspaceSize=1G
```

Comment:
**Question**: Why was this added? (not that it's wrong just wondering)

Reply:
I was occasionally running into out-of-memory errors while building the project. Weren't you or someone else also running into this, and bumped up the memory in a PR?